src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
-        "Air Elemental",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#descriptiveCard#"
+    ],
+
+    "descriptiveCard": [
+        "an altered #realCard#",
+        "an artist-signed #realCard#",
+        "an HP coffee-stained #realCard#",
+        "a counterfeit #realCard#",
+        "a foil #realCard#",
+        "a foreign-language #realCard#",
+        "a From the Vaults #realCard#",
+        "a proxy #realCard#",
+        "#realCard.a# borrowed from a dealer",
+        "#pro#'s personal #realCard#"
+    ],
+
+    "cardAlone": [
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#fakeCard#"
+    ],
+
+    "realCard": [
```

Comment:
Disagree on the weighting--fake cards should be a rarity, even among the tweets that use 2 card names. 10% should be their absolute max IMO, but even a touch lower like here I think is preferable.